### PR TITLE
Set up testing environment

### PR DESCRIPTION
**Pull request by tzavidas**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -34,6 +34,39 @@
...
+            <goal>install</goal>
+          </goals>
+        </execution>
+      </executions>
+      <configuration>
+        <version>1.9.8</version>
+          <skip>${js-tests.skip}</skip>
+      </configuration>
+      </plugin>
+
```
Comment (maxwbuckley) [inline pom.xml:53]: Remove empty line? There is no empty line before the previous plugin

**Review by maxwbuckley — APPROVED** [review]

File: src/test/MIT.LICENSE [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2008-2019 Pivotal Labs
```
Comment (semenovm) [inline src/test/MIT.LICENSE:1]: Does this file belong here?

File: src/test/lib/jasmine-3.6.0/boot.js [hunk 3]
```diff
@@ -0,0 +1,139 @@
+/**
```
Comment (semenovm) [inline src/test/lib/jasmine-3.6.0/boot.js:1]: I believe jasmine-maven-plugin should download it automatically.

File: pom.xml [hunk 4]
```diff
@@ -34,6 +34,39 @@
...
+          <goals>
+            <goal>install</goal>
+          </goals>
+        </execution>
+      </executions>
+      <configuration>
+        <version>1.9.8</version>
+          <skip>${js-tests.skip}</skip>
+      </configuration>
+      </plugin>
```
Comment (semenovm) [inline pom.xml:52]: Do you need this dependency, I thought you will use headless chrome instead?

File: src/test/SpecRunner.html [hunk 5]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```
Comment (semenovm) [inline src/test/SpecRunner.html:1]: Is this file still required if you are using jasmin-plugin?

**Review by semenovm — CHANGES_REQUESTED** [review]

File: src/test/MIT.LICENSE (hunk 2 above)
Comment (maxwbuckley) [reply src/test/MIT.LICENSE:1]: :D

**Pull request closed without merging**